interfaces/HO-Portal/src/assets/i18n/en.json

```
@@ -245,7 +245,7 @@
         "pageTitle": "Program JSON"
       },
       "bulk-import": {
-        "import-error": "Something went wrong with the import. Check the required extension (.csv) and format of the CSV-file.",
+        "import-error": "Something went wrong with the import. <ul><li>Check the required extension (.csv)</li><li>Check the format and column-names of the CSV-file</li><li>The 'paymentAmountMultiplier' column should contain only whole numbers >= 1</li></ul>",
```

I think we should split this up into separate translatable sentences and format them into a list in the template.
Having this type of HTML in the translation-process is very annoying/complex.

```suggestion
        "import-error": {
          "intro": "Something went wrong with the import.",
          "check": {
            "extension": "Check the required extension (.csv)",
            "format": "Check the format and column-names of the CSV-file",
            "column": {
              "paymentAmountMultiplier": "The 'paymentAmountMultiplier' column should contain only whole numbers >= 1",
              "anotherFutureFeature": "The X-column requires Y-values"
            }
          }
        },
```

Something like that...

---

can you make this change?

---

We'll pick it up in a future PR.